jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/TimePickerAssert.java

```diff
@@ -8,216 +8,161 @@
+        jdiAssert(actualHours, Matchers.is(hours), errorMsg);
         return this;
     }
 
-    @JDIAction("Assert that all '{name}' enabled hours/minutes are correct")
-    public TimePickerAssert properEnabledHoursOrMinutes(List<String> enabledHoursOrMinutes) {
-        Timer.waitCondition(() -> element().getEnabledHoursOrMinutes().equals(enabledHoursOrMinutes));
-        List<String> actualEnabledHoursOrMinutes = element().getEnabledHoursOrMinutes();
-        jdiAssert(actualEnabledHoursOrMinutes, Matchers.containsInAnyOrder(enabledHoursOrMinutes.toArray()),
-                String.format("Actual enabled hours/minutes '%s' are not equal to expected '%s'",
-                        actualEnabledHoursOrMinutes, enabledHoursOrMinutes));
+    @JDIAction("Assert that '{name}' in title are '{0}'")
+    public TimePickerAssert minutes(int minutes) {
+        int actualMinutes = element().titleMinutes();
+        String errorMsg = format("Expecting '%s' hours in title, but was '%s'", actualMinutes, minutes);
+        jdiAssert(actualMinutes, Matchers.is(minutes), errorMsg);
         return this;
     }
 
-    @JDIAction("Assert that all '{name}' disabled hours/minutes are correct")
-    public TimePickerAssert properDisabledHoursOrMinutes(List<String> disabledHoursOrMinutes) {
-        Timer.waitCondition(() -> element().getDisabledHoursOrMinutes().equals(disabledHoursOrMinutes));
-        List<String> actualDisabledHoursOrMinutes = element().getDisabledHoursOrMinutes();
-        jdiAssert(actualDisabledHoursOrMinutes, Matchers.containsInAnyOrder(disabledHoursOrMinutes.toArray()),
-                String.format("Actual disabled hours/minutes '%s' are not equal to expected '%s'",
-                        actualDisabledHoursOrMinutes, disabledHoursOrMinutes));
+    @JDIAction("Assert that '{name}' in title are '{0}'")
+    public TimePickerAssert seconds(int seconds) {
+        int actualSeconds = element().titleSeconds();
+        String errorMsg = format("Expecting '%s' hours in title, but was '%s'", actualSeconds, seconds);
+        jdiAssert(actualSeconds, Matchers.is(seconds), errorMsg);
         return this;
     }
 
-    @JDIAction("Assert that '{name}' has twelve hour format")
-    public TimePickerAssert twelveHourFormat() {
-        jdiAssert(element().getAllHoursElements().size(), Matchers.equalTo(12), "Element has not " +
-                "12 hours format");
+    @JDIAction("Assert that '{name}' has '{0}' disabled numbers")
+    public TimePickerAssert disabledNumbers(Integer... expectedDisabledNumbers) {
+        List<Integer> actualDisabledNumbers = element().disabledClockNumbers();
+        String errorMsg = format("Expecting '%s' numbers to be disabled. But disabled numbers were '%s'",
+            Arrays.asList(expectedDisabledNumbers), actualDisabledNumbers);
+        jdiAssert(actualDisabledNumbers, Matchers.containsInAnyOrder(expectedDisabledNumbers), errorMsg);
```

Comment:
Not clear that we are checking only partial contains for values, typically we need to change all disabled values.

Reply:
Good point. Missed this - it would work only for hours, as all hours are shown on the clock. But will not work for seconds/minutes. And I don't see a solution for now. Probably this should be extracted to a separate story/feature.